src/bp-performance/bp-performance-loader.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * BuddyBoss Performance Loader.
+ *
+ * A performance component, Allow to cache BuddyBoss Platform REST API.
+ *
+ * @package BuddyBoss\Performance\Loader
+ * @since BuddyBoss 1.6.0
+ */
+
+// Exit if accessed directly.
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Set up the bp-performance component.
+ *
+ * @since BuddyBoss 1.6.0
+ */
+function bp_setup_performance() {
+	buddypress()->performance = new BP_Performance_Component();
+}
+add_action( 'bp_setup_components', 'bp_setup_performance', 1 );
```

Comment:
@KartikSuthar can't we need to comment on this also as we don't want to show components

Reply:
@dipskakadiya We have not called components, So this file was not loaded at all.